bin/local-dev/wordpress/Dockerfile

```diff
@@ -92,3 +92,8 @@ COPY config/php/* /usr/local/etc/php/conf.d/
 # Setup xdebug.
 RUN pecl install xdebug; \
 	docker-php-ext-enable xdebug;
+
+# Instal WP CLI
+RUN curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
```

Comment:
Could we add this as a Composer development dependency instead? That way we're locking it down to a specific version and avoiding surprises as things change. This will probably require a dedicated Composer script added to `composer.json` since the local vendor directory is not included in the `$PATH` of the container.

Reply:
That's a very good idea actually! Thanks!